src/configuration/app-containers.md

```diff
@@ -306,30 +301,56 @@ It has a few subkeys listed below:
     example regular expressions that you could use to provide rules:
     *\\.css$,\\.js$,\\.gif$,\\.jpe?g$,\\.png$,\\.tiff?$,\\.wbmp$,\\.ico$,\\.jng$,\\.bmp$,\\.svgz?$,\\.midi?$,\\.mpe?ga$,\\.mp2$,\\.mp3$,\\.m4a$,\\.ra$,\\.weba$,\\.3gpp?$,\\.mp4$,\\.mpe?g$,\\.mpe$,\\.ogv$,\\.mov$,\\.webm$,\\.flv$,\\.mng$,\\.asx$,\\.asf$,\\.wmv$,\\.avi$,\\.ogx$,\\.swf$,\\.jar$,\\.ttf$,\\.eot$,\\.woff$,\\.otf$,/robots\\.txt$*.
 
-*Example*
+### [Example] A basic `web` block for PHP
 
 ```yaml
 web:
     locations:
-        "/":
-            root: "public"
-            passthru: "/index.php"
+        '/':
+            root: 'public'
+            passthru: '/index.php'
             index:
                 - index.php
+            # No caching for static files.
+            # (Dynamic pages use whatever cache headers are generated by the program.)
             expires: -1
             scripts: true
             allow: true
             rules:
+                # Disallow .mp4 files specifically.
                 \.mp4$:
                     allow: false
                     expires: -1
         # Set a 5 min expiration time for static files here; a missing URL
-        # will passthru to the "/" location above and hit the application
+        # will passthru to the '/' location above and hit the application
         # front-controller.
-        "/sites/default/files":
+        '/images':
             expires: 300
             passthru: true
+            allow: false
+            rules:
+                # Only allow static image files from the images directory.
+                '\.(jpe?g|png|gif|svgz?|ico|bmp)$':
+                    allow: true
+```
+
+### [Example] Advanced rewrite rules
```

Comment:
@sruggier Is this block superseding https://docs.platform.sh/configuration/app-containers.html#url-rewriting?